internal/dag/extension_processor.go

```diff
@@ -110,6 +118,11 @@ func (p *ExtensionServiceProcessor) buildExtensionService(
 		ClientCertificate:  clientCertSecret,
 	}
 
+	if len(p.MinimumTLSVersion) != 0 {
+		extension.MinimumProtocolVersion = p.MinimumTLSVersion
+		extension.CipherSuites = p.CipherSuites
+	}
```

Comment:
I think per @sunjayBhatia's previous comment we should handle these independently, so:

```go
if len(p.MinimumTLSVersion) != 0 {
	extension.MinimumTLSVersion = p.MinimumTLSVersion
}
if len(extension.CipherSuites) != 0 {
	extension.CipherSuites = p.CipherSuites
}
```